### ISSUE TITLE
No syntax highlighting code blocks are still highlighted

### ISSUE THREAD

**Issue by fmaussion** [executablebooks/sphinx-book-theme#522]
### Describe the bug

The code blocks without language indication are wrongly highlighted.

### Reproduce the bug

See: https://jupyterbook.org/reference/cheatsheet.html#code-and-syntax-highlighting

![image](https://user-images.githubusercontent.com/10050469/157124775-697bffca-f478-436a-96e0-36a233bc3bba.png)

**Comment by pradyunsg** [comment]
This is behaviour from Sphinx -- if you want to have no default highlighting, you'll want to set the language as "```none".

Alternatively, you can set `html_language` in the conf.py (https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-highlight_language).

**Comment by pradyunsg** [comment]
That said, that example in jupyter-book likely needs to be fixed!

**Comment by choldgraf** [comment]
Good catch! I'll move this one to the Jupyter Book repo, since it's unique to the docs there
